### ISSUE TITLE
9th slide should use `None` in the scala example

### ISSUE THREAD

**Issue by sakoht** [koeninger/scala-view-of-rust#1]
To make an equivalent comparison between Scala and Rust, this line on slide 9:
`var y: Option[Int] = _`

Should read:
`var y: Option[Int] = None`

Like it does in Rust.  And it will not blow up.

**Comment by koeninger** [comment]
No, they aren't equivalent, that's the point. Options in scala can still be
null.

**Comment by sakoht** [comment]
Ah so in Rust I can't set an option to equal `ptr::null()`?

It seemed like that was the Rust equivalent to foolishly assigning `null` instead of `None` to an `Option` in Scala.
